MOBILE_CLIENT/COMMON/_MEDIUM/XML_INJECTION/meta.json

```diff
@@ -0,0 +1 @@
+{"risk_rating": "high", "short_description": "XML Injection is a type of attack where an attacker injects malicious XML code into an application, which can then be executed by the application.", "references": {"External Entity Injection (OWASP)": "https://owasp.org/www-community/vulnerabilities/XML_External_Entity_(XXE)_Processing", "XPath Injection (OWASP)": "https://owasp.org/www-community/attacks/XPATH_Injection"}, "title": "XML Injection", "privacy_issue": false, "security_issue": true, "categories": {}}
```

Comment:
The description is not correct. Executed by the application?!

Reply:
Well isn't the point of those PRs to demonstrate how well AI can perform in KB generation with minimal interference from our side? if I fix everything before sending my PRs you wouldn't know if it was the AI that generated that content or me who added it manually.

Reply:
No, absolutely not.